core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -240,31 +278,37 @@ object ZkUtils extends Logging {
   /**
    *  make sure a persistent path exists in ZK. Create the path if not exist.
    */
-  def makeSurePersistentPathExists(client: ZkClient, path: String) {
+  def makeSurePersistentPathExists(client: ZkClient, path: String, acls: List[ACL] = DefaultAcls) {
```

Comment:
Do we want to have a default parameter of `DefaultAcls` here? Is it right that we would want to be sure to pass this parameter (excluding tests)?